### PR TITLE
Build improvements for the inkscape1.0 branch

### PR DESCRIPTION
**Pull request by rumpeltux**

### CODE REVIEW

File: lib/extensions/auto_satin.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import inkex
 
-from ..elements import SatinColumn, Stroke
+from ..elements import SatinColumn
```
Comment (kaalleen) [inline lib/extensions/auto_satin.py:5]: Why do you remove the import of Stroke here? It's needed in line 62...

**Review by kaalleen — CHANGES_REQUESTED** [review]

File: lib/extensions/params.py [hunk 2]
```diff
@@ -525,7 +525,7 @@ def tab_sort_key(tab):
 
                 # If multiple tabs are enabled, make sure dependent
                 # tabs are grouped with the parent.
-                parent,
+                parent and parent.name,
```
Comment (lexelby) [inline lib/extensions/params.py:528]: Hi!  Thanks for your PR, we really appreciate your contribution.

What is this change about?  Use it not working as just `parent`?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -19,6 +19,27 @@ jobs:
...
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
```
Comment (lexelby) [inline .github/workflows/build.yml:42]: Thank you for figuring this stuff out.  I never did manage to find the time to figure out caching.

File: lib/extensions/params.py (hunk 2 above)
Comment (rumpeltux) [reply lib/extensions/params.py:528]: Yes, this fixes the error mentioned in https://github.com/inkstitch/inkstitch/issues/980#issuecomment-762686051

File: .github/workflows/build.yml (hunk 3 above)
Comment (rumpeltux) [reply .github/workflows/build.yml:42]: Actually this is just copy and paste of the examples from https://github.com/actions/cache/blob/main/examples.md#node---yarn

But it does seem to work :)

File: lib/extensions/auto_satin.py (hunk 1 above)
Comment (rumpeltux) [reply lib/extensions/auto_satin.py:5]: Oops, this must have been introduced by some merge issue. Thanks for catching. Should be fixed now.

**Pull request merged**